ethereum-types/Cargo.toml

```diff
@@ -1,17 +1,17 @@
 [package]
 name = "ethereum-types"
-version = "0.8.0"
+version = "0.8.1"
```

Comment:
@ordian are you saying this must be `0.9` too?

Reply:
yes, because it exposes `fixed_hash::H256` from 0.6 instead of `fixed_hash::H256` from 0.5